charts/hedera-mirror-rest/values.yaml

```
@@ -60,13 +60,13 @@ hpa:
   behavior: {}
   enabled: true
   maxReplicas: 15
-  metrics:
-    - type: Resource
-      resource:
-        name: cpu
+  metrics: # Using a custom metric exposed via the prometheus adapter. TODO: Monitor and change the averageValue as per use.
+      pods:
+        metric:
+          name: requests_per_second
         target:
-          type: Utilization
-          averageUtilization: 80
+          type: AverageValue
+          averageValue: 300000m
```

Can drop `m` unit after dividing by 1000.
```suggestion
          averageValue: 300
```